frontend/app/(droplets)/d/[slug]/page.tsx

```
@@ -19,7 +19,7 @@ import { getCurrentUser } from "@/lib/auth/session";
 import { getAuthorizedUserByEmail } from "@/lib/requests/authorized-user";
 import { getEnrollmentsByAuthorizedUser } from "@/lib/requests/enrollment";
 import { StarRating } from "@/components/ui/rating-stars";
-import { getDropletAverageRating } from "@/lib/requests/enrollment";
+import { calculateDropletAverageRating } from "@/lib/requests/enrollment";
```

ESLint

[error] 'calculateDropletAverageRating' is defined but never used.

(@typescript-eslint/no-unused-vars)